session-signatures/getLitActionSessionSigs/src/index.ts

```diff
@@ -48,9 +48,16 @@ export const getSessionSigsLitAction = async () => {
       `✅ Minted new PKP with public key: ${pkp.publicKey} and ETH address: ${pkp.ethAddress}`
     );
 
-    const litActionCode = `(async () => {
-      Lit.Actions.setResponse({ response: "true" });
-    })();`;
+    console.log("🔄 Adding example permitted Lit Action to the PKP");
+    await litContracts.addPermittedAction({
+      ipfsId: "QmTaYbqnGwrmseoDQdTNoU9FNzaiaTpKApgMFWqbsWs4Cr",
```

Comment:
My only concern with using a IPFS CID is if it's no longer resolvable in the future when a user tries to run this code. Maybe we use `litActionCode` for this example and comment out the IPFS code with a comment saying this is how you'd do it. This was the example should always work

Reply:
Ever since the change was made that verifies if a PKP has the action permitted, I think the `litActionCode` has become somewhat useless. If you try to run it with only the `litActionCode`, the scoping error occurs. Because it needs to be added as a permitted action, it needs to be somewhere on the IPFS because adding it requires the `ipfsId`. 

Using `litActionCode` in `getLitActionSessionSigs` will only work if that code is first on the IPFS.

Reply:
Okay, I think we should bring this up to the rest of the team for discussion on whether or not this is intended

For this PR, who is pinning the code on IPFS, did you use Pinata?

Reply:
I think the lit action was uploaded through the Lit Explorer:

https://explorer.litprotocol.com/ipfs/QmTaYbqnGwrmseoDQdTNoU9FNzaiaTpKApgMFWqbsWs4Cr